vyper/types/convert.py

```diff
@@ -96,11 +99,20 @@ def to_bytes32(expr, args, kwargs, context):
 
 
 def convert(expr, context):
-    output_type = expr.args[1].s
+
+    if isinstance(expr.args[1], ast.Name):
+        output_type = expr.args[1].id
+    else:
+        raise ParserException(
```

Comment:
What do you think about a deprecation warning instead?

Reply:
yeah sure, why not ;)